app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/OfflineDatabaseService.kt

```diff
@@ -248,12 +248,35 @@ class OfflineDatabaseService internal constructor(
                 firebaseUser.uid,
                 User.Type.FIREBASE,
                 firebaseUser.displayName,
-                firebaseUser.email
+                firebaseUser.email,
+                getUser(User.Type.FIREBASE, firebaseUser.uid)!!.isAdmin,
```

Comment:
I suggest using the ?. operator and return the non-updated user if getUser() returns null in combination with logging that the operation failed

Reply:
Thanks for suggesting that, I was looking for an alternative

Reply:
Please adress this comment